### ISSUE TITLE
Dynamic workspace names.

### ISSUE THREAD

**Issue by gaurav712** [swaywm/sway#4013]
How can I make workspace names dynamic? What I mean by that : 
![sway](https://user-images.githubusercontent.com/45332121/55430298-58da9780-55ab-11e9-92d7-e254d0e43a7a.png)

I want workspaces to look something like **1>2>3>...** but the background color for the "U+E0B0" is an issue (I want them to be like the status bar on right).
I want them to change the background color for  "U+E0B0" automatically when focus toggles on the workspace next to it.

I want this :
![right](https://user-images.githubusercontent.com/45332121/55431914-3fd3e580-55af-11e9-947b-58bcd35a9bfd.png)

not this : 
![wrong](https://user-images.githubusercontent.com/45332121/55431941-4cf0d480-55af-11e9-8bc5-11fadf8ad00c.png)

**Comment by pvreevil** [comment]
Swaybar? I noticed some weird spacing between workspaces, how are you dealing with this and the borders? 

White bg is pango markup.
![15543131013212843](https://user-images.githubusercontent.com/46428150/55500206-359d0000-561e-11e9-9b79-c916b6c2de24.png)

**Comment by gaurav712** [comment]
You can find all my dots in the dotfiles repo : https://github.com/gaurav712/dotfiles_sway

Btw, I think it's because in my case, bg and border colors are same

**Comment by pvreevil** [comment]
I meant, even using the same color the border will add extra space, i wonder if powerline ws would be possible without gaps.

**Comment by gaurav712** [comment]
Oh, I don't have that problem. If you wan't status's source. It's here : https://github.com/gaurav712/super_simple_projects/blob/master/status.c

**Comment by pvreevil** [comment]
> I wan't kind-of same stuff on left. That shouldn't be a problem as it's fine on right.

but there is a problem, look at this weird padding at ws name.

![imageedit_10_9375594992](https://user-images.githubusercontent.com/46428150/55508416-3d65a000-5630-11e9-85cb-24f6537bcead.png)

` bindsym $mod+9 workspace 9 <span font='mononoki Nerd Font 10'>9</span>`
`bindsym $mod+0 workspace 10 <span font='mononoki Nerd Font 10'>10</span>`

**Comment by pvreevil** [comment]
> ```
> `where program asks sway for focused workspace and if it's $ws1, sends "<span>..</span>" for current workspace and previous (if any) as well.`

I think sway must be reloaded every time the ws names are changed to make effect.

**Comment by gaurav712** [comment]
> > I wan't kind-of same stuff on left. That shouldn't be a problem as it's fine on right.
> 
> but there is a problem, look at this weird padding at ws name.
> 
> ![imageedit_10_9375594992](https://user-images.githubusercontent.com/46428150/55508416-3d65a000-5630-11e9-85cb-24f6537bcead.png)`
 But it won't be the case if you use all unicode characters. I'll use font awesome or maybe emojis.

**Comment by gaurav712** [comment]
Oh and by the way, I noticed you're using gentoo.  How's sway there withoud systemd ? I'm thinking about switching back.

**Comment by gaurav712** [comment]
> 
> I think sway must be reloaded every time the ws names are changed to make effect.

I don't think so. Because it can't switch workspaces without refreshing the bar. So, It'll be redundant in my opinion.

**Comment by pvreevil** [comment]
> Oh and by the way, I noticed you're using gentoo. How's sway there withoud systemd ? I'm thinking about switching back.

running perfectly under elogind

**Comment by pvreevil** [comment]
> But it won't be the case if you use all unicode characters as they'll have same spacing. I'll use font awesome or maybe emojis.
> 

I meant the right and left padding are swaybar stuff, regardless of ws title content and size.
